crypto/curve25519/curve25519.c

```diff
@@ -4520,13 +4521,9 @@ void GFp_x25519_sc_muladd(uint8_t *s, const uint8_t *a, const uint8_t *b,
   s[31] = (uint8_t)(s11 >> 17);
 }
 
-
-/* Prototype to avoid -Wmissing-prototypes warnings. */
-void GFp_ed25519_scalar_mask(uint8_t a[32]);
-
-void GFp_ed25519_scalar_mask(uint8_t a[32]) {
+void GFp_x25519_scalar_mask(uint8_t a[32]) {
```

Comment:
> Also, I suggest we name it `GFp_curve25519_scalar_mask` since it works for both Ed25519 and X25519.

Agreed— there are some other functions with a `x25519` prefix that are used in the Ed25519 code. Would it be best to change these names too, in one separate commit?